source/virtualBuffers/gecko_ia2.py

```diff
@@ -26,6 +26,9 @@ def _normalizeControlField(self,attrs):
 		ariaCurrent = attrs.get("IAccessible2::attribute_current")
 		if ariaCurrent != None:
 			attrs['current']= ariaCurrent
+		placeholder = self._getPlaceholderAttribute(attrs, "IAccessible2::attribute_placeholder")
+		if placeholder != None:

```

Comment:
This would be better as `if placeholder` or, if you want to be super explicit, `if placeholder is not None`. I see `!= None` happens for `ariaCurrent` as well, so probably worth fixing that one too.